### ISSUE TITLE
Create opponent boolean in settings

### ISSUE THREAD

**Issue by bryankennedy** [cns-iu/xmacroscope#106]
Some locations don't have opponent competition. Add a field in the settings to allow this to be switched off.

The Run logic should handle this, routing the start screen around the opponent selection screens.

**Comment by bryankennedy** [comment]
This setting would be used across all the client systems to update functionality.

**Comment by bryankennedy** [comment]
I'm moving this to Phase 2 for now. We'll be completely striping opponents out of the Phase 1 system.

**Comment by bherr2** [comment]
Why even keep Opponents in then?

**Comment by bryankennedy** [comment]
I agree. I have some leftover opponent stuff in the data layer. I'll pull it out for now.

Once we build this out at SMM we will need the opponent since it's baked into our experience.
